xc/device.ml

```diff
@@ -1408,6 +1416,8 @@ module Vusb = struct
     if device_model = Profile.Qemu_trad then
       raise (Internal_error (Printf.sprintf "Failed to plug VUSB %s because domain %d uses device-model profile %s." id domid (Profile.string_of device_model)));
     debug "Vusb plugged: vusb device %s plugged" id;
+    (* Need to reset USB device before passthrough to vm according to CP-24616 *)
+    call_usb_reset [ hostbus ^ "-" ^ hostport ];
```

Comment:
I'm not sure but: `call_usb_reset` could also take a single string as argument as this is the use case. This could be turned into a list by the function before it is passed to `forkexecd`. This would make it bit simpler.